addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -496,6 +520,13 @@ func set_room(value: Node2D) -> void:
 #endregion
 
 #region Private ####################################################################################
+# Increments the usage count for the specified command
+func _increment_command_count(command_id: int) -> void:
+	if not _command_usage_count.has(command_id):
+		_command_usage_count[command_id] = 0
```

Comment:
Maybe you could use `_command_usage_count.get_or_add(command_id, 0)` instead?

Reply:
I didn't know about that :scream: terrific!